contracts/contracts/strategies/BaseConvexMetaStrategy.sol

```diff
@@ -66,11 +66,13 @@ abstract contract BaseConvexMetaStrategy is BaseCurveStrategy {
         address[] calldata _pTokens,
         InitConfig calldata initConfig
     ) external onlyGovernor initializer {
-        require(_assets.length == 3, "Must have exactly three assets");
+        require(
+            _assets.length == CURVE_BASE_ASSETS,
+            "Must have exactly three assets"
```

Comment:
nit: Error message isn't generic

Reply:
this is used by the old OUSD AMO strategies which use the 3Pool as the base pool to the Curve Metapool. I kept AMO changes to a minimum as there's a separate PR that refactors the AMO strategies.